### ISSUE TITLE
Add `--fast` flag to skip Markdown validation

### ISSUE THREAD

**Issue by hukkinj1** [executablebooks/mdformat#50]
See https://github.com/executablebooks/mdformat/pull/49#issuecomment-702740026

Add a CLI flag for skipping Markdown validation.
I would name the flag `--fast` or `--skip-validation`.

**Comment by KyleKing** [comment]
Do you have a preference for which flag to add? I may have time to submit a PR by next week?

**Comment by hukkin** [comment]
I think `--unsafe` and `--skip-validation` are both alright.

I always thought if things are eventually stable for years, it may be possible to change the default value of this check to off. If that happens we need the opposite flag. Then we'd have either `--safe` or `--no-skip-validation` out of which `--safe` seems much better. But that'd translate to `safe = true/false` in TOML which with `--unsafe` would slightly break the rule where CLI opt naming directly corresponds with TOML attribute naming....

Would `--no-validate` and `validate = false/true` be horrible? :smile: Either that or just `--unsafe` and `unsafe = true/false`, and don't worry about the future. Whoever implements can decide :smile:

**Comment by KyleKing** [comment]
I like `--no-validate`, but I would probably default to `validate = true` because plugins often break the AST (or maybe that's just me 😆 https://github.com/search?q=org%3AKyleKing+renders+to+different+HTML&type=issues)

**Comment by hukkin** [comment]
> I would probably prefer to default to validate = true

Yeah right now the default must definitely not change. And if it ever changes it may be useful for experimental plugins to be able to keep the validation on, similar to how they can turn validation off via `CHANGES_AST` already.

**Comment by hukkin** [comment]
Closed by https://github.com/hukkin/mdformat/pull/496
